#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_validate_kasumi.desktop.dpatch <Rolf Leggewie <foss@rolf.leggewie.biz>>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: validate the desktop file and improve description

@DPATCH@

--- orig/kasumi.desktop.in	2006-06-03 16:30:07.000000000 +0200
+++ new/kasumi.desktop.in	2009-08-02 05:21:42.000000000 +0200
@@ -1,15 +1,16 @@
 [Desktop Entry]
 Version=1.0
-Encoding=UTF-8
 Name=Anthy Dictionary editor
 Name[ja]=Anthy 辞書管理
-GenericName=Management tool for a private dictionary of anthy
+GenericName=Management tool for anthy private dictionary
 GenericName[ja]=Anthy 個人辞書管理ツール
 Comment=Edit Anthy dictionary.
 Comment[ja]=Anthy の個人辞書に単語を登録、削除、編集するツールです。
 Exec=kasumi
-Icon=kasumi.png
+Icon=kasumi
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Applications;Settings;
+Categories=Settings;
+Keywords=Japanese,Dictionary
+Keywords[ja]=日本語,辞書
